mongodb/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# MongoDB was last available in the 3.9 release
+FROM alpine:3.9
```

Comment:
3.9 was version of MongoDB or alpine image?

Reply:
alpine:3.9 is the version of alpine. I added another comment about this so you can see the package listing.